code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -365,7 +365,26 @@ var/veil_thickness = CULT_PROLOGUE
 				to_chat(user, "<span class='sinister'>You have trouble focusing, things will go bad if you keep using your blood.</span>")
 			else if (blood_before > BLOOD_VOLUME_SURVIVE && blood_after < BLOOD_VOLUME_SURVIVE)
 				to_chat(user, "<span class='sinister'>It will be all over soon.</span>")
-			H.take_overall_damage(data["user_amount"] ? 0.1 : 0)
+			H.take_overall_damage(data[BLOODCOST_AMOUNT_USER] ? 0.1 : 0)
 
 	data["blood"] = blood
 	return data
+
+
+/obj/item/proc/get_cult_power()
+	return 0
+
+/mob/proc/get_cult_power()
+	var/power = 0
+	var/list/valid_slots = list(
```

Comment:
I've seen other collabs request for such lists that are just there as a pseudo-define to be defined as var/static/list so they don't need to keep being recreated each time a proc like this is called. Could you do so for this?